examples/preprocessing/plot_run_ica.py

```diff
@@ -23,8 +23,11 @@
 
 ###############################################################################
 # Read and preprocess the data. Preprocessing consists of:
+#
 # - MEG channel selection
+#
```

Comment:
As @larsoner pointed out, I think blank lines are only required before and after a list - not in between the list items...